app/src/main/java/ch/epfl/sdp/blindly/profile_setup/ProfileShowMe.kt

```diff
@@ -42,15 +38,12 @@ class ProfileShowMe : AppCompatActivity() {
             val intent = Intent(this, ProfilePassions::class.java)
             val showMe = findViewById<RadioButton>(radioGroup.checkedRadioButtonId)
                     .text.toString().toLowerCase().capitalize()
```

Comment:
I've already made that remark last time, but Cirrus doesn't seem to be happy about the `text.capitalize()` and prefers `capitalize(text)`. There seems to be another problem with the locale.

Reply:
Yep, I know but as I've answered Lollipoke, I tried but none of these two function can be called statically (String.toLowerCase(show_me) doesn't exist))